lib/python/src/bailo/helper/entry.py

```diff
@@ -46,11 +46,18 @@ def update(self) -> None:
 
         logger.info(f"ID %s updated locally and on server.", self.id)
 
-    def card_from_schema(self, schema_id: str) -> None:
+    def card_from_schema(self, schema_id: str | None = None) -> None:
         """Create a card using a schema on Bailo.
 
-        :param schema_id: A unique schema ID
+        :param schema_id: A unique schema ID, defaults to None. If None, either minimal-general-v10 or minimal-data-card-v10 is used
         """
+        if schema_id is None:
+            if self.kind == EntryKind.MODEL:
+                schema_id = MinimalSchema.MODEL
+            if self.kind == EntryKind.DATACARD:
+                schema_id = MinimalSchema.DATACARD
+            warnings.warn(f"Schema ID not provided, will attempt to default to {schema_id}.")
```

Comment:
This shouldn't be a warning it's expected behaviour.

Reply:
Done